assets/src/features/shared/hooks/useSmartphoneViewport.tsx

```diff
@@ -12,21 +12,23 @@ const useSmartphoneViewport = (): ScreenInfo => {
 
   useEffect(() => {
     if (typeof window !== "undefined") {
+      const isLandscape =
+        screen.orientation?.type.includes("landscape") || window.matchMedia("(orientation: landscape)").matches;
```

Comment:
This seems like something very specific for one browser/os/device. Can you please add a brief comment explaining the reason for this change?